package.json

```diff
@@ -4,9 +4,12 @@
   "description": "ADLDAP Federation Connector",
   "main": "server.js",
   "scripts": {
+    "lint:check": "npx eslint admin/**/*.js connector-setup/**/*.js lib/** test/**/*.js",
+    "lint:fix": "npx eslint admin/** connector-setup/** lib/** test/** --fix",
```

Comment:
Shouldn't link:fix also filter the *.js files. Can we change it to "lint:fix": "npm run lint:check -- --fix"

Reply:
I'll do a second pass on this repo to fix all the linter issues -- leaving this for now.